### PR TITLE
`unplugin-typia` to be main feature of setup guide docs

### PR DESCRIPTION
**Pull request by samchon**
@ryoppippi I've decided to make `unplugin-typia` to be main feature of setup guide docs right now.

 The reason why not waiting for its stablization is, the stabitlization can be boosted by feedbacks from `typia` users.

Therefore, needs your review.

### CODE REVIEW

File: website/pages/docs/setup.mdx [hunk 1]
```diff
@@ -242,183 +200,115 @@ yarn prepare
...
-    null !== input &&
-    "string" === typeof input.id &&
-    $is_uuid(input.id) &&
-    "string" === typeof input.email &&
-    $is_email(input.email) &&
-    "number" === typeof input.age &&
-    19 <= input.age &&
-    100 >= input.age
-  );
+```javascript filename="next.config.mjs" copy showLineNumbers
```
Comment (ryoppippi) [inline website/pages/docs/setup.mdx:283]: I wrote more detailed version of config for Next.js. so please replace it.
https://jsr.io/@ryoppippi/unplugin-typia/doc/next/~/default

File: website/pages/docs/setup.mdx [hunk 2]
```diff
@@ -242,183 +200,115 @@ yarn prepare
...
+Currently, `unplugin-typia` supports the following bundlers:
+  - [Vite](https://vitejs.dev/)
+  - [Rollup](https://rollupjs.org/)
+  - [esbuild](https://esbuild.github.io/)
+  - [webpack](https://webpack.js.org/)
+  - [Next.js](https://nextjs.org/)
+  - [Rspack](https://www.rspack.dev/)
+  - [Rolldown](https://rolldown.rs/)
+  - [farm](https://farm-fe.github.io/)
+
```
Comment (ryoppippi) [inline website/pages/docs/setup.mdx:238]: - [bun.build](https://bun.sh/docs/bundler)

I also added a [partial support for Bun](https://jsr.io/@ryoppippi/unplugin-typia/doc/bun/~).
It works for generating code, but Bun fails to solve the CJS codes including `randexp.js` (like https://github.com/oven-sh/bun/issues/7465)
So It's up to you if you include `Bun` to the official document.

I think this plugin for `Bun.build` will be the easiest way to try typia in the future, but we need to wait for Bun's stablization

**Review by ryoppippi — COMMENTED** [review]
Thanks! @samchon 
More or less, This is OK.
I think my `unplugin-typia` is getting stable(and actually I implemented cache feature too!)

I wrote a couple of comments, so please check them out!!

File: website/pages/docs/setup.mdx (hunk 2 above)
Comment (ryoppippi) [reply website/pages/docs/setup.mdx:238]: I updated a Docs.
We can use `Bun.build` fully. So please add this !

**Comment by samchon** [comment]
@ryoppippi Updated as you wish. If you want to modify or add more things, please send a new PR.

**Comment by ryoppippi** [comment]
Thank you!!! @samchon

**Pull request merged**